CONTRIBUTING.md

```diff
@@ -9,6 +9,6 @@ The process is straight-forward.
  - Write the code for your device, notification service, sensor, or IoT thing.
  - Ensure tests work.
  - Create a Pull Request against the [**dev**](https://github.com/home-assistant/home-assistant/tree/dev) branch of Home Assistant.
-
+Click here
```

Comment:
This addition doesn't make sense.

Reply:
For Hacktoberfest one only needs opened PRs, not have to be merged. We can stop it by adding the invalid label. Indicate they are trying to game the system.